src/test/java/uk/gov/hmcts/reform/blobrouter/reconciliation/report/ReconciliationMapperTest.java

```diff
@@ -0,0 +1,159 @@
+package uk.gov.hmcts.reform.blobrouter.reconciliation.report;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.common.io.Resources;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import uk.gov.hmcts.reform.blobrouter.config.ServiceConfiguration;
+import uk.gov.hmcts.reform.blobrouter.config.StorageConfigItem;
+import uk.gov.hmcts.reform.blobrouter.data.reconciliation.statements.model.EnvelopeSupplierStatement;
+
+import java.io.IOException;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+
+import static com.google.common.io.Resources.getResource;
+import static java.nio.charset.StandardCharsets.UTF_8;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.BDDMockito.given;
+import static org.mockito.Mockito.mock;
+
+@SuppressWarnings("unchecked")
+class ReconciliationMapperTest {
+
+    private ReconciliationMapper reconciliationMapper;
+    private ServiceConfiguration serviceConfiguration = mock(ServiceConfiguration.class);
+    private ObjectMapper objectMapper = new ObjectMapper();
+
+    private Map<String, StorageConfigItem> storageConfig = mock(Map.class);
+
+    @BeforeEach
+    void setUp() {
+        given(serviceConfiguration.getStorageConfig()).willReturn(storageConfig);
+        reconciliationMapper =
+            new ReconciliationMapper(objectMapper, serviceConfiguration);
+    }
+
+    @Test
+    void should_filter_by_target_container() throws IOException {
+        // given
+        setupStorageConfig();
+        String content = Resources.toString(
+            getResource("reconciliation/valid-supplier-statement.json"),
+            UTF_8
+        );
+
+        LocalDate reportDate = LocalDate.now();
+        EnvelopeSupplierStatement envelopeSupplierStatement = new EnvelopeSupplierStatement(
```

Comment:
Remember you can use `var` now.
No need to state 3 times that it's "envelope supplier statement".